srcpkgs/openldap/template

```diff
@@ -15,16 +15,17 @@ configure_args="--prefix=/usr
  --disable-static --with-yielding_select=yes --enable-modules"
 hostmakedepends="pkg-config groff automake libtool"
 makedepends="libressl-devel libsasl-devel db-devel libltdl-devel"
+depends="openldap-tools>=${version}_${revision}"
 conf_files="/etc/openldap/ldap.conf /etc/openldap/slapd.conf"
 short_desc="OpenLDAP (Lightweight Directory Access Protocol)"
 license="OpenLDAP License v2.8 - BSD alike"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="http://www.openldap.org"
-distfiles="ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-$version.tgz"
+distfiles="ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-${version}.tgz"
 checksum=cdd6cffdebcd95161a73305ec13fc7a78e9707b46ca9f84fb897cd5626df3824
 
 system_accounts="ldap"
-ldap_homedir="/var/lib/openldap"
+ldap_homedir=/var/lib/openldap
```

Comment:
Why unquoting this?

Reply:
it shouldn't be, my formalizing script should be ignoring fixed strings on username related variables like system_accounts and _homedir.